### PR TITLE
Quality of Life: short import path for components

### PR DESCRIPTION
**Pull request by hubzie**
* Apply comments from previous PR (#8)
* Import components with `import {...} from '@components'`

### CODE REVIEW

**Comment by katzper01** [comment]
have you researcher if there is a way to make a custom eslint rule for this? to throw errors if we use relative path instead of '@components'?

**Comment by hubzie** [comment]
Good catch, fixed

**Review by katzper01 — CHANGES_REQUESTED** [review]
I left two comments

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "axios": "^1.3.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.9.0"
+    "react-router-dom": "^6.9.0",
+    "tsconfig-paths-webpack-plugin": "^4.0.1"
```
[the remark this answers is not in the document]
Comment (hubzie) [reply package.json]: Fixed

**Review by katzper01 — APPROVED** [review]
Approved

**Pull request merged**